downloads/swan-lake-release-notes/swan-lake-beta2/RELEASE_NOTE.md

```diff
@@ -1,42 +1,213 @@
+
+**Previous Syntax**
+
+```ballerina
+stream<record {}, error> resultStream = 
+jdbcClient->query(`SELECT * FROM Customers`,Customer);
+
+stream<Customer, sql:Error> customerStream = 
+        <stream<Customer, sql:Error>>resultStream;
+```
+
+According to the previous syntax, you can give the `Customer` record as the expected record type of the stream but you cannot get that stream directly. To do that, you have to cast the returned stream to the expected stream. Therefore, this has been improved to directly access data as an expected record type.
+
+**New Syntax**
+
+```ballerina
+stream<Customer, error> customerStream = sqlClient->query(`SELECT * FROM Customers`);
+```
+
+#### `http` Package
+
+- Updated to respond with a 202 Accepted response when the resource function returns nil. If the `http:Caller` is used in the resource, then, returning nil from the resource leads to a 500 Internal Server Error response.
+- Updated to log the error stack trace when an error is returned from the resource function and on connection failures.
+- Updated to log a warning when the same request is responded more than one time.
+
+#### `graphql` Package
+
+- Made the GraphQL resource execution non-blocking.
+- Updated to send a `BAD_REQUEST` status code for the responses with document validation errors.
+
+#### `ftp` Package
+
+- Changed the `get`, `append`, and `put` method APIs to support `stream` instead of using `io:ReadableByteChannel` for reading and writing files.
+- Updated to support the FTP protocol with password and private key based authentications.
+
+#### `websub` Package
+
+- Updated to log the error stack trace when an error is returned from the remote function of the subscriber service.
+- Updated to return module-specific errors from the WebSub public APIs.
+- Updated to allow non-remote methods in the subscriber service.
+
+#### `websubhub` Package
+
+- Updated to log the error stack trace when an error is returned from the remote function of the hub service.
+- Updated to return the module-specific errors from the WebSubHub public APIs.
+- Updated to allow non-remote methods in the Hub Service.
+- Updated to allow `http:Headers` as an optional parameter in specific remote methods (`onRegisterTopic`/`onDeregisterTopic`/`onUpdateMessage`/`onSubscription`/`onUnsubscription`).
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta2%22+label%3AType%2FBug).
+
+## Code to Cloud Updates
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22).
+
+## Developer Tools Updates
+
+### New Features
+
+#### Debugger 
+- Added expression evaluation support for client remote method call actions.
```

Comment:
Full stop at the end of bullet sentence seems inconsistent. Please check all the places.

Reply:
We have used full stops in all the bullet points.